joss/paper.md

```diff
@@ -1,51 +1,120 @@
 ---
-title: '`Eureka!`: An Exoplanet Transit Spectroscopy Pipeline'
+title: '`Eureka!`: An End-to-End Pipeline for JWST Time-Series Observations'
 tags:
+  - Python
   - JWST
-  - python
+  - HST
   - astronomy
   - exoplanets
   - spectroscopy
+  - photometry
 authors:
-
-  - name: Eureka Collaboration 
+  - name: Taylor J. Bell
+    orcid: 0000-0003-4177-2149
+    affiliation: 1
+  - name: Eva-Maria Ahrer
+    orcid: 0000-0003-0973-8426
+    affiliation: 2
+  - name: Jonathan Brande
+    orcid: 0000-0002-2072-6541
+    affiliation: 3
+  - name: Aarynn L. Carter
+    orcid: 0000-0001-5365-4815
+    affiliation: 4
+  - name: Adina D. Feinstein
+    orcid: 0000-0002-9464-8101
+    affiliation: 5
+  - name: Giannina Guzman
+    orcid: 0000-0001-6340-8220
+    affiliation: 6
+  - name: Megan Mansfield
+    orcid: 0000-0003-4241-7413
+    affiliation: 7
+  - name: Sebastian Zieba
+    orcid: 0000-0003-0562-6750
+    affiliation: 8
+  - name: Caroline Piaulet
+    orcid: 0000-0002-2875-917X
+    affiliation: 9
+  - name: Joseph Filippazzo
+    orcid: 0000-0002-0201-8306
+    affiliation: 10
+  - name: Erin M. May
+    orcid: 0000-0002-2739-1465
+    affiliation: 11
+  - name: Kevin B. Stevenson
+    orcid: 0000-0002-7352-7941
+    affiliation: 11
   - name: Laura Kreidberg
     orcid: 0000-0003-0514-1147
-    affiliation: X
+    affiliation: 8
 affiliations:
- - name: Max-Planck-Institut für Astronomie, Königstuhl 17, D-69117 Heidelberg, Germany
-   index: X
-date: xxx
+  - name: BAER Institute, NASA Ames Research Center, Moffet Field, CA 94035, USA
+    index: 1
+  - name: Centre for Exoplanets and Habitability, University of Warwick, Gibbet Hill Road, CV4 7AL Coventry, UK
```

Comment:
Please change "Centre for Exoplanets and Habitability" to "Department of Physics", everything else is correct - thank you!